core/vcpu.c

```diff
@@ -448,7 +448,7 @@ struct vcpu_t *vcpu_create(struct vm_t *vm, void *vm_host, int vcpu_id)
             sizeof(struct vcpu_state_t), HAX_MEM_NONPAGE);
     if (!vcpu->state)
         goto fail_4;
-    memset(vcpu->state, 0, sizeof(struct vcpu_state_t));
+    memset(vcpu->state, 0, sizeof(struct vcpu_state_t)); //cr8 = 0
```

Comment:
This comment is unclear (without looking at the whole commit).

Reply:
I will remove it in the final version. This is the place where cr8 is initialized. I used it to analyze the cr8 support feature.